metadata-ingestion/src/datahub/cli/ingest_cli.py

```diff
@@ -233,6 +233,31 @@ def parse_restli_response(response):
     return rows
 
 
+@ingest.command()
```

Comment:
I'm a bit nervous about taking the top-level default option on this one even though I like the convenience of it. 

Why should
```
datahub ingest <something>
```

default to ingesting an MCP file or directory with MCP files?

versus:
```
datahub ingest --source file <path>
```

Reply:
You use this as `datahub ingest metadata-file my_file.json`

```
$ datahub ingest metadata-file --help
Usage: datahub ingest metadata-file [OPTIONS] PATH

  Ingest from a metadata json file or directory of files.

  This requires that you've run `datahub init` to set up your config.

Options:
  --help  Show this message and exit.
```